apps/src/lib/levelbuilder/script-editor/ScriptEditor.jsx

```diff
@@ -95,6 +95,7 @@ class ScriptEditor extends React.Component {
     initialShowCalendar: PropTypes.bool,
     initialWeeklyInstructionalMinutes: PropTypes.number,
     isMigrated: PropTypes.bool,
+    includeStudentFacingLessonPlans: PropTypes.bool,
```

Comment:
To help keep the difference between the state and the props clear we have been calling things that are also in the state initialIncludeStudentFacingLessonPlans